test/unit/containers/sprite-selector-item.test.jsx

```diff
@@ -66,4 +67,10 @@ describe('SpriteSelectorItem Container', () => {
         expect(global.confirm).toHaveBeenCalled();
         expect(onDeleteButtonClick).not.toHaveBeenCalled();
     });
+
+    test('Has font regexp works', () => {
+        expect('font-family="Sans Serif"'.match(HAS_FONT_REGEXP)).toBeTruthy();
+        expect('font-family="none"'.match(HAS_FONT_REGEXP)).toBeFalsy();
+        expect('font-family="none" font-family="Sans Serif"'.match(HAS_FONT_REGEXP)).toBeTruthy();
```

Comment:
It might be a good idea to add things like this:
```js
'fake-font-family="none"'.match(HAS_FONT_REGEXP)
'fake-font-family="some"'.match(HAS_FONT_REGEXP)
'font-family-fake="none"'.match(HAS_FONT_REGEXP)
'font-family-fake="some"'.match(HAS_FONT_REGEXP)
```

Reply:
What results do you expect for those 4 lines?
I changed your regex a bit to match "font-family" instead of 'font-family="..."', since I think SVG allows adding spaces between words. So currently you would get
no match
match
match
match
(Incidentally, do you know if we should consider capitalization?)
None of these need to match, but false positives are better than missing legitimate cases, and I don't think we have to worry too much about fake-font-family attributes unless.... there were someone on the inside manipulating storage.....🤔

Reply:
I was thinking there might be some SVG editors out there that add attributes like `myeditor-alternate-font-family` or `font-family-myeditor-enhanced` or something. I don't know if that happens or if it's even allowed by SVG though... maybe this is a non-issue?

Reply:
The font injecting code looks for the "font-family" attribute specifically, so if there were attributes like "myeditor-alternate-font-family" and we did parse the SVG, it still wouldn't inject those fonts. Do you mean maybe it should?

Reply:
Probably not. I was just thinking that the regex should match those cases where the font injecting code actually makes a change, so probably all of these cases should return false?

Reply:
ohh I see. So
(at least 1 whitespace)font-family(any whitespace)=(any whitespace)"(Not none)
?
I need a block-based programming language for regex

Reply:
Hmm, that's a good point. Maybe it's better to err on the side of false positives (like you said 1.3 million years ago) and refine if we find that this is a performance problem.